service/history/replication/task_processor.go

```diff
@@ -429,11 +431,16 @@ func (p *taskProcessorImpl) processSingleTask(replicationTask *types.Replication
 			tag.TaskType(request.TaskInfo.GetTaskType()),
 			tag.Error(err),
 		)
-		//TODO: uncomment this when the execution fixer workflow is ready
-		//if err = p.triggerDataInconsistencyScan(replicationTask); err != nil {
-		//	p.logger.Warn("Failed to trigger data scan", tag.Error(err))
-		//	p.metricsClient.IncCounter(metrics.ReplicationDLQStatsScope, metrics.ReplicationDLQValidationFailed)
-		//}
+
+		// Uncommenting this to verify that the scanner indeed gets triggered.
+		//Adding extra flipr protection since this is an ongoing test.
+		if p.config.EnableDataScan(request.TaskInfo.GetDomainID()) {
```

Comment:
There is a shard cache with domain names in task processor: p.shard.GetDomainCache().GetDomainName(domainID)

Reply:
thanks for the pointer. Will update the flipr to domain Name.